### DISCUSSION TITLE
有个问题……

### DISCUSSION THREAD

**Discussion by WFGoose** [share121/inter-knot#2643] [其他]
这个绳网等级是和github活跃天数挂钩的吗？我好像只有之前做Java作业和同学用过几次，现在是2级

**Comment by 00zzxx** [comment]
等级和公开的库挂钩吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by zhixuan45** [comment]
我github上有13库，自建的有8库但是等级只有11
![image](https://github.com/user-attachments/assets/afb3a64d-8007-4e28-8244-e18b1da22406)
